src/Peering/Peering/help/Update-AzPeering.md

```diff
@@ -0,0 +1,274 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Peering.dll-Help.xml
+Module Name: Az.Peering
+online version: https://docs.microsoft.com/en-us/powershell/module/az.peering/update-azpeering
+schema: 2.0.0
+---
+
+# Update-AzPeering
+
+## SYNOPSIS
+Sets the Peering. Use this Command in conjunction with `Set-AzDirectPeeringConnectionObject` or `Set-AzExchangePeeringConnectionObject`.
+
+## SYNTAX
+
+### ParameterSetNameDefaultExchange (Default)
+```
+Update-AzPeering -InputObject <PSPeering> [[-ExchangeConnection] <PSExchangeConnection[]>]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### ParameterSetNameDefaultDirect
+```
+Update-AzPeering -InputObject <PSPeering> [-UseForPeeringService <Boolean>]
+ [-DirectConnection <PSDirectConnection[]>] [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
+
+### ParameterSetNameByResourceIdDirect
+```
+Update-AzPeering -ResourceId <String> [-UseForPeeringService <Boolean>]
+ [-DirectConnection <PSDirectConnection[]>] [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
+
+### ParameterSetNameByResourceIdExchange
+```
+Update-AzPeering -ResourceId <String> [-ExchangeConnection] <PSExchangeConnection[]>
```

Comment:
ExchangeConnection should be optional everywhere.

Reply:
no we have to have an exchange connection for ParameterSetByRGandName and ResourceId. If we dont have that, what are we updating? Nothing else is editable.